src/doc/src/reference/unstable.md

```diff
@@ -1230,66 +1230,134 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
-[dependencies]
-log.workspace = true
-log2.workspace = true
-```
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
 
-Example 2:
-```toml
-# in workspace's Cargo.toml
 [workspace.package]
 version = "1.2.3"
 authors = ["Nice Folks"]
 description = "..."
 documentation = "https://example.github.io/example"
-readme = "README.md"
-homepage = "https://example.com"
-repository = "https://github.com/example/example"
-license = "MIT"
-license-file = "./LICENSE"
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+Keys that are supported:
+`authors`, `categories`, `description`, `documentation`, `edition`,
+`exclude`, `homepage`, `include`, `keywords`, `license`, `license-file`,
+`publish`, `readme`, `repository`, `rust-version`, and`version`.
+
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. Inheriting dependencies was added as a
+way to reduce duplication of dependencies across members of a workspace.
+Inheriting a dependency is as simple as setting the dependency in the workspace
+under `[workspace.dependencies]` as you would in`[dependencies]`. From there
+[specify that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
+Full example:
+
+```toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
+[workspace.dependencies]
+log = "0.3.1"
+serde = { git = 'https://github.com/serde-rs/serde' }
+wasm-bindgen-cli = { path = "crates/cli" }
+```
+
+```toml
+# [PROGECT_DIR]/bar/Cargo.toml
+[project]
+name = "bar"
+version = "0.2.0"
+
+[dependencies]
+log.workspace = true
+serde.workspace = true
+wasm-bindgen-cli.workspace = true
+```
+
+Dependencies in `[workspace.dependencies]` cannot be declared as `optional`.
+The `optional` key must be omitted or, if present, must be `false`. Dependencies
+in the `[dependencies]`, `[dev-dependencies]`, `[build-dependencies]`, and
+`[target."...".dependencies]` sections support the ability to reference the
+`[workspace.dependencies]` definition of dependencies.
+
+[inheriting-a-dependency-from-a-workspace]: #inheriting-a-dependency-from-a-workspace
+[workspaces]: workspaces.md
+[the-workspacemetadata-table]: workspaces.md#the-workspacemetadata-table
+
+### Inheriting a dependency from a workspace
+
+This would be in [`specifying-dependencies.md`][specifying-dependencies],
+under [Renaming dependencies in Cargo.toml][renaming-dependencies-in-cargotoml]
```

Comment:
Should this go earlier in the document where its talking about dependency sources?

In some ways, I see this filling that role but it also goes into a lot more detail than just sources and covers content from other sections in this document, making it fit at the end as well

Reply:
I thought about putting it in sources but decided against it. It also seems "more advanced" than just a source is why I put it where it is. I could go either way on this